src/pages/docs/Rating.jsx

```diff
@@ -10,6 +10,16 @@ export class Rating extends React.Component {
     };
   }
 
+  static getDerivedStateFromProps = (nextProps, prevState) => {
```

Comment:
is this a method of the React.Component?

Reply:
Yes, see https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops